.github/workflows/build-subsystems.yaml

```diff
@@ -0,0 +1,114 @@
+name: Build Subsystems
+
+on:
+  workflow_call:
+    inputs:
+      dry-run:
+        description: "An indication of whether to commit/publish results"
+        required: true
+        type: string
+      is-dependabot:
+        description: "An indication of a dependabot pull request"
+        required: true
+        type: string
+
+jobs:
+  setup:
```

Comment:
I don't like this, but now at least we know where to change it if/when the assumptions no longer hold :)

Reply:
I don't understand